code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -98,6 +98,14 @@
 	if(emagged == 2) //Emag functions
 		if(istype(loc,/turf/simulated))
 
+			for(var/mob/living/carbon/victim in src.loc)

```

Comment:
what procs specifically? i asked around in coderbus and this is how they said to do it.
good point on the src use though, i'll fix that later.